### PR TITLE
@dzucconi - fills out feed feature, adds periodic animation and interim holding page

### PR DESCRIPTION
**Pull request by broskoski**
Going to leave off the animated gif so you will check the beauty of this feature in all its glory. This is a first pass at the automatic feed queueing. Since the entries will be exhausted every cycle it was much less complicated than your queuing method (renders part of the collection, animates the items to their proper place, pops a model off the top and renders the over the completed animation).

Definitely looking for input on how I could get more test coverage here!

### CODE REVIEW

File: app/views/feed.coffee [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-View      = require '../core/view'
-Queue     = require '../collections/queue'
-Loadable  = require '../utils/loadable'
-Entries   = require '../collections/entries'
+config          = require '../config/config'
+View            = require '../core/view'
+Queue           = require '../collections/queue'
+Loadable        = require '../utils/loadable'
+Entries         = require '../collections/entries'
+Tags            = require '../collections/tags'
```
Comment (dzucconi) [inline app/views/feed.coffee]: omg just remove the spaces

Comment (broskoski) [inline app/views/feed.coffee]: I kinda like them though :|

File: test/views/feed.coffee [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        @view.entries.length.should.equal 8
+        @view.maybeShowHolder()
+        @view.entries.length.should.equal 7
+
+      it 'Shows holding page when entry list is exhausted', ->
+        @view.entries = new Entries @entries[0]
+        @view.entries.length.should.equal 1
+        @view.maybeShowHolder()
+        @transitionSpy.called.should.be.ok
+
```
Comment (dzucconi) [inline test/views/feed.coffee]: Yay tests

**Comment by dzucconi** [comment]
As far as more test coverage: I wouldn't worry about covering any of the animation stuff with specs; just break out as much logic into the collections/models and test those.

**Pull request merged**